tinygrad/runtime/ops_python.py

```diff
@@ -82,18 +85,20 @@ def __call__(self, *bufs, global_size:Tuple[int,int,int]=(1,1,1), local_size:Tup
           loop_ends[idp[0]] = i
           i = idp[0]
           continue
-        elif uop is UOps.BARRIER:
+        # TODO: implement UOps.IF and UOps.ENDIF
+        elif uop in (UOps.BARRIER, UOps.IF, UOps.ENDIF):
           # in the python emulator, the warp is always in sync
           i += 1
           continue
+
         assert dtype is not None, f"{uop} is missing a dtype"
         dl[i] = dtype
         if uop is UOps.DEFINE_GLOBAL:
           assert dtype.fmt is not None
           ul[i] = [pbufs.pop(0).cast(dtype.fmt)] * warp_size
         elif uop is UOps.DEFINE_LOCAL:
           assert dtype.fmt is not None
-          lbuf = memoryview(bytearray(arg[1]*dtype.sz))
+          lbuf = memoryview(bytearray(arg[1]*dtype.itemsize))
```

Comment:
This fix is definitely correct, would merge this alone right away.

Reply:
Fixes commits are mostly independent, so I can cherry-pick this and push it as a separate PR if you want to. What about other fixes?